tests/proxy_models/models.py

```diff
@@ -205,3 +206,12 @@ class Improvement(Issue):
 class ProxyImprovement(Improvement):
     class Meta:
         proxy = True
+
+
+class AuthUserProxy(AuthUser):
```

Comment:
This is testing a change in `contrib.auth` so I think the test would be better in `auth_tests`.

Reply:
Does it work if you explicitly set the `app_label`?

```
class Bar(models.Model):
    name = models.CharField(max_length=255)

    class Meta:
        app_label = 'a_different_app'
```